LICENSE

```diff
@@ -252,54 +252,6 @@ APPENDIX B: Additional licenses relevant to this product:
 
 
 
-    =============================================================
-    Google Protobuf License:
```

Comment:
Protobuf was used to test the UDF-8 code. Now that that has been removed, this license is no longer needed.